solutions/admin.py

```diff
@@ -0,0 +1,4 @@
+from django.contrib import admin
+
+# Register your models here.
+from .models import SolutionHistory, Message
```

Comment:
❗ import뿐만 아니라 admin 사이트에 등록해야 합니다.
https://www.notion.so/Code-Convention-276cc780cd7d806ea542fe3b9748257d#276cc780cd7d80718c4fe20425b6e1ba

```suggestion
from .models import SolutionHistory, Message

admin.site.register(SolutionHistory)
admin.site.register(Message)
```